kotsadm/web/src/components/apps/Dashboard.jsx

```diff
@@ -524,6 +536,22 @@ class Dashboard extends Component {
     })
   }
 
+  onSnapshotOptionChange = (selectedSnapshotOption) => {
+    this.setState({ selectedSnapshotOption });
+  }
+
+  onSnapshotOptionClick = () => {
+    const { selectedSnapshotOption } = this.state;
+
+    if (selectedSnapshotOption.option === "full") {
+      this.startASnapshot(true);
+    } else if (selectedSnapshotOption.option === "partial") {
+      this.startASnapshot(false);
```

Comment:
I think we could make this method a little smaller and easier to read buy doing something like this 

```
if (selectedSnapshotOption.option === "learn") {
    window.open("https://kots.io/vendor/snapshots/overview/", "_blank"); 
} else {
    this.startASnapshot(selectedSnapshotOption.option);
}
```

And then up in the `startASnapshot` method we pass `snapshotType` as the param and have the `isFull` logic there

```const isFull = snapshotType === "full";```

To me this is more readable because you're passing the type instead of the bool. As it stands now passing the true/false, my brain reads this as "Start a snapshot? Yes/No".